components/Comment/Comment.tsx

```diff
@@ -0,0 +1,68 @@
+import CommentHeader from "./CommentHeader";
+import CommentReadOnly from "./CommentReadOnly";
+import { css, StyleSheet } from "aphrodite";
+import CommentActions from "./CommentActions";
+import colors from "~/config/themes/colors";
+import { Comment as CommentType } from "./lib/types";
+import { useState } from "react";
+import CommentEditor from "./CommentEditor";
+
+type CommentArgs = {
+  comment: CommentType;
+  handleUpdate: Function;
+  handleCreate: Function;
+};
+
+const Comment = ({ comment, handleUpdate, handleCreate }: CommentArgs) => {
+  const [isReplyOpen, setIsReplyOpen] = useState(false);
+  const [isEditMode, setIsEditMode] = useState(false);
```

Comment:
Can we type these to be Boolean? 

We've run into issues before where if this changes to a number, specifically 0, then the below 

```
isReplyOpen && ...
```

Will display a 0 on the website

Reply:
Sure, I didn't think it was needed to type case primitives but this makes sense